src/components/common/ListRow/index.js

```diff
@@ -0,0 +1,37 @@
+import { Fragment } from 'react';
+import cn from 'classnames';
+
+import './styles.css';
+
+const ListRow = ({
+  key,
```

Comment:
You don't need to get the `key` here and pass it down. React already knows when you pass it to `ListRow`